azure-pipelines.yml

```diff
@@ -36,8 +40,8 @@ stages:
       - job: Windows_NT
         pool:
           ${{ if eq(variables._RunAsInternal, True) }}:
-            name: NetCoreInternal-Pool
-            queue: BuildPool.Server.Amd64.VS2019
+            name: NetCore1ESPool-Internal
```

Comment:
this will need to use the -Svc pool (probably best through the pool-providers.yml)

Reply:
Thanks, that's a good point